doc/source/contributor/environments/ci-multinode.rst

```diff
@@ -324,3 +324,67 @@ with:
 
 There are various other options for sonobuoy, see the `documentation
 <https://sonobuoy.io/docs/>`_ for more details.
+
+Wazuh
+======
+
+Wazuh is supported but not deployed by default. If you are using the standard
+[StackHPC multinode
+terraform](https://github.com/stackhpc/terraform-kayobe-multinode), there is a
+``deploy_wazuh`` terraform variable that will add it to the automated setup.
+
+This guide will explain how to add Wazuh to an existing multinode environment.
```

Comment:
Make it clear that this is done automatically by the script when using terraform?

Reply:
My intention was that the [previous paragraph](https://github.com/stackhpc/stackhpc-kayobe-config/pull/552/files#diff-6c737a95f8f4930660d780c0727646e75c4834280d2515b617d87855e0ea61e8R331-R334) would explain that it can be done automatically with terraform, but this was an alternative. I can reword it to make it more clear if you think that would be best.

Reply:
It's clear if you read it carefully... How about a subheading to separate it from the first paragraph?

Reply:
I've separated the sections out so now we have two subheadings, one for new deployments, one for existing